rclcpp/CMakeLists.txt

```diff
@@ -619,8 +619,12 @@ if(BUILD_TESTING)
     target_link_libraries(test_wait_set ${PROJECT_NAME})
   endif()
 
+  set(append_library_dirs "${CMAKE_CURRENT_BINARY_DIR}")
+  if(WIN32)
+    set(append_library_dirs "${append_library_dirs}/$<CONFIG>")
+  endif()
```

Comment:
This is already set [here](https://github.com/ros2/rclcpp/blob/master/rclcpp/CMakeLists.txt#L510-L513). The test only needs to use it.

Reply:
:+1:  I missed that the first time around.

Reply:
My mistake as well, fixing.